test/Integration/test-csx/Common/Product.csx

```diff
@@ -234,4 +242,66 @@ public class ProductUtilities
         }
         return products;
     }
+}
+
+public static class Utils
```

Comment:
Would suggest making a separate file for this, we should generally be keeping 1 class/file. 

But also - can you not just use the one from the Utils class directly with the #load directive?